src/items/extern-crates.md

```diff
@@ -51,11 +59,13 @@ need for specifying `extern crate`.
 > Beginning in the 2018 edition, [use declarations] can reference crates in
 > the extern prelude, so it is considered unidiomatic to use `extern crate`.
 
-> **Note**: Additional crates that ship with `rustc`, such as [`proc_macro`],
-> [`alloc`], and [`test`], currently aren't available in the extern prelude
-> and must be brought into scope with an `extern crate` declaration, even in
-> the 2018 edition. `use` paths must reference the `extern crate` item (such
-> as using [`crate::`] or [`self::`] path prefixes).
+> **Note**: Crates not explicitly named with the `--extern` flag with `rustc`
+> are not included in the extern prelude. This means that additional crates
+> that ship with `rustc`, such as [`proc_macro`], [`alloc`], and [`test`],
+> currently aren't available in the extern prelude and must be brought into
+> scope with an `extern crate` declaration, even in the 2018 edition. `use`
+> paths must reference the `extern crate` item (such as using [`crate::`] or
+> [`self::`] path prefixes).
```

Comment:
The last sentence is obsolete with `extern_crate_item_prelude`.
Imports *may* of course reference `extern crate my_crate` with `use crate::my_crate` or `use self::my_crate`, but it's not necessary, `use my_crate` will work too.